### DISCUSSION TITLE
What's the difference between a blinko and notes?

### DISCUSSION THREAD

**Discussion by meldarionqeusse** [blinkospace/blinko#112] [Q&A] [answered]
As per the title. Looked into the docs but did not find anything relevant to my question

**Comment by dnburgess** [comment]
Came to ask this same question. Would really like to know the difference.

**Comment by zezretro** [comment]
Yup me too, whats the difference?

**Reply by dnburgess** [reply]
> Yup me too, whats the difference?

Found this on their website after not finding it here:

> "Blinko" section is crafted for capturing quick flashes of inspiration. Users can jot down spontaneous ideas instantly without worrying about format or complex operations. These records can be set to automatically clear at intervals, ensuring that the content doesn’t pile up excessively, keeping the section clean and efficient.
> 
> "Notes" section provides a more organized management system where users can categorize notes with tags and utilize batch operations to easily organize and classify a large volume of notes.

**Comment by baldemar-wuda** [comment]
I would like to have the option to disable "blinkos" altogether, as I see no difference between them and notes (they just add to the confusion, and additional clicks back to notes).

**Comment by LiangWei88** [comment]
目前我的用法是用 blinko 做间歇日记, 就是做了些啥, 准备要做啥之类, 一些要备案留档的想法和资料放在笔记里面

---
My current usage is to use blinko to keep an intermittent diary, that is, what I have done, what I am going to do, etc., and some ideas and information that need to be filed are placed in the notes

**Comment by zezretro** [comment]
Thank you for your input. I feel I am still unclear about what is the difference between a note a blinko.

Could you explain more?

thank you

**Comment by LiangWei88** [comment]
其实我也没有特别清楚作者原来的用意, 不过我的用法是将 blinko 作为 Interstitial Journaling 的用途, 其他笔记放在 note 里面.
相关概念参考: https://betterhumans.pub/replace-your-to-do-list-with-interstitial-journaling-to-increase-productivity-4e43109d15ef

**Comment by blinko-space** [comment, marked as answer]
https://blinko.mintlify.app/how-to-use/blinko-or-note#note-long-form-content-and-knowledge-base

This document explaining what difference about blinko and note

**Reply by LiangWei88** [reply]
> https://blinko.mintlify.app/how-to-use/blinko-or-note#note-long-form-content-and-knowledge-base
> 
> This document explaining what difference about blinko and note

已经看过, 应该理解了, 还需要大量的使用实践去感受两者的区别

**Comment by nachbelichtet** [comment]
The "Blinks" are brilliant in my opinion. Most of my notes are temporary. In Obsidian I gave them tags like "1week" or "6month"
This determined the period after which they would presumably become irrelevant. I was able to filter by tags and current time and either delete the notes or convert them into permanent notes.

Perhaps such a “best before date” would also be useful for Blinks?
